homework_weekend/js/app.js

```diff
@@ -0,0 +1,85 @@
+// question 1
+
+
+// parametes are the names listed in the function definitinon and agruments are the real
+//reall values passed to the function
+
+// b. a console prints out the function and a return returns the value of the function
+
+// c. Scope determines the accessibility of variables, objects, and functions from 
+//different parts of the code.   
+
+function checkPalindrome(str); 
+var arrayOfLetters = str.split("");
+arrayOfLetters = arrayOfLetters.reverse();
+var str2 = arrayOfLetters.join("");
+
+
+if checkPalindrome(str)	{
+	var checkPalindrome = true;
+
+}
+
```

Comment:
you need parentheses around the condition of the if statement.  also what are you trying to do on lines 18-21?  it's not exactly clear. adding a comment describing what you're trying to do can help you get the code you want.

Reply:
you've got a lot of the pieces you need for this function; you just need to put them together.